lms/djangoapps/courseware/management/commands/tests/test_dump_course.py

```diff
@@ -55,15 +55,19 @@ def load_courses(self):
         # Add a course with a unicode name, if the modulestore
         # supports adding modules.
         if hasattr(store, 'create_xmodule'):
```

Comment:
@cpennington and actually, I was able to remove the hasattr check entirely because mixed modulestore has the create_xblock method and calls the default store. Even when the XML modulestore is one of the options, it is not the default store.